code/lib/core-client/src/preview/start.ts

```diff
@@ -35,12 +35,8 @@ interface ClientAPIFacade {
   /* deprecated */
   addParameters: (...args: any[]) => never;
   /* deprecated */
-  clearDecorators: (...args: any[]) => never;
-  /* deprecated */
   addLoader: (...args: any[]) => never;
   /* deprecated */
-  setAddon: (...args: any[]) => never;
-  /* deprecated */
   getStorybook: (...args: any[]) => never;
   /* deprecated */
   storiesOf: (...args: any[]) => never;
```

Comment:
Should we remove all these?

Reply:
In theory we should remove everything except for `storiesOf`.

In practice, we should NOT remove these addParameters etc. until we remove `storiesOf`, presumably in 8.0.

That's going to be a big project, potentially with complex automigrations, etc. Out of scope for now.

Reply:
@tmeasday do you agree?

Reply:
I think we can remove all the global ones `addParameters` etc as they have the alternative `export const parameters` now.

AFAIK `storiesOf` is the only one that doesn't have a modern alternative.

Reply:
TBH I'm not 100% convinced we need an automigration for a feature that has been deprecated for multiple years now with a simple upgrade path, but I can be convinced.